src/cswinrt/code_writers.h

```diff
@@ -1472,7 +1472,17 @@ remove => %;
                 },
                 [&](std::string_view type_name)
                 {
-                    w.write("^@\"%\"", type_name);
+                    std::string sanitized_type_name;
```

Comment:
It looks like the result of the above test class is `    [global::TestComponentCSharp.MyString(Content = @"a string with embedded quotes, like the string \""quotes\""")]
`.  I am wondering whether we should keep the \ in addition to double quote or whether the \ should be excluded when replacing it with a double quote.  Thoughts?  The change as-is is technically fine in that it no longer will result in a compiler error.

Reply:
the `\` is coming from the string in the original IDL, so we shouldn't touch it. IIRC there are requirements around how strings with embedded quotes are processed by the C preprocessor before passing it to MIDL that require the use of the backslash

Reply:
Right, to escape a quote in C / MIDL, you would use a / like /".  Now we are projecting that into C# which it really is the equivalent of "" when using verbatim strings with @ .   But I can see trying to do that running into weird issues with edge cases and given this is only in attributes, I am fine with leaving it as how you have it.